ivre/db/mongo.py

```diff
@@ -2028,6 +2041,12 @@ def parse_args(self, args, flt=None):
             flt = self.flt_and(flt, self.searchsource(args.source))
         if args.timeago is not None:
             flt = self.flt_and(flt, self.searchtimeago(args.timeago))
+        if args.id is not None:
```

Comment:
It seems that `args.id` can still be `None` even with the use of `nargs="+"`, should I write
`if args.id:` instead ?